src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartIndenterTests.cs

```diff
@@ -2717,12 +2717,11 @@ static void M()
     }
 }
 ";
-            // Need to confirm expected behavior after discard/deconstruction parsing changes
-            // https://github.com/dotnet/roslyn/issues/15813
+
             AssertSmartIndent(
                 code,
                 indentationLine: 6,
-                expectedIndentation: 8);
+                expectedIndentation: 12);
```

Comment:
Should this indentation be 8 or 12?

Reply:
It's been this way for more than 5 years without a complaint, so seems like we have implicit agreement.